extensions/sql-migration/src/wizard/sqlSourceConfigurationPage.ts

```diff
@@ -0,0 +1,108 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as os from 'os';
+import { MigrationWizardPage } from '../models/migrationWizardPage';
+import { MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
+import * as constants from '../constants/strings';
+import { createLabelTextComponent, createHeadingTextComponent } from './wizardController';
+
+export class SqlSourceConfigurationPage extends MigrationWizardPage {
+	private _view!: azdata.ModelView;
+
+	constructor(wizard: azdata.window.Wizard, migrationStateModel: MigrationStateModel) {
+		super(wizard, azdata.window.createWizardPage(constants.SOURCE_CONFIGURATION, 'MigrationModePage'), migrationStateModel);
+	}
+
+	protected async registerContent(view: azdata.ModelView): Promise<void> {
+		this._view = view;
+		const form = view.modelBuilder.formContainer()
+			.withFormItems(
+				[
+					await this.createSourceCredentialContainer(),
+				]
+			);
+		await view.initializeModel(form.component());
+	}
+
+	public async onPageEnter(): Promise<void> {
+	}
+	public async onPageLeave(): Promise<void> {
+	}
+	protected async handleStateChange(e: StateChangeEvent): Promise<void> {
+	}
+
+	private async createSourceCredentialContainer(): Promise<azdata.FormComponent> {
+
+		const connectionProfile = await this.migrationStateModel.getSourceConnectionProfile();
+
+		let username;
+		switch (connectionProfile.authenticationType) {
+			case 'SqlLogin':
+				username = connectionProfile.userName;
+				this.migrationStateModel._authenticationType = 'SqlAuthentication';
+				break;
+			case 'Integrated':
+				username = os.userInfo().username;
```

Comment:
Have you tested this on all OSes?

For non-Windows OSes this name doesn't really mean anything either since for those it's based on your kerberos setup. Is this what design wanted you to do?

Reply:
I am assuming that integrated auth is not available for other OSes.

Reply:
Nope, that's what kerberos auth is for : https://docs.microsoft.com/en-us/sql/azure-data-studio/enable-kerberos?view=sql-server-ver15